clarity_ext/domain/container.py

```diff
@@ -62,13 +62,22 @@ def create(repr):
         Creates a ContainerPosition from different representations. Supported formats:
             "<row as A-Z>:<col as int>"
             "<row as int>:<col as int>"
+            "<row as A-Z><col as int>"
             (<row>, <col>) where both are integers
-            (<row>, <col>) where column is a string (e.g. A)
+            (<row>, <col>) where row is a string (e.g. A)
         """
```

Comment:
Normally, I think that there are a little too much "if-else" statements in a row in the create() method. It affect how hard/easy it is the read the method. Now, it doesn't matter. I will merge this.

Reply:
Yeah, I agree.